orml/vesting/src/mock.rs

```diff
@@ -3,114 +3,85 @@
 #![cfg(test)]
 
 use super::*;
-use frame_support::{
-	construct_runtime, parameter_types,
-	traits::{ConstU64, Everything},
-};
+use frame_support::traits::ConstU64;
+use frame_support::{construct_runtime, derive_impl, parameter_types};
 use frame_system::RawOrigin;
-use sp_core::H256;
-use sp_runtime::{traits::IdentityLookup, BuildStorage};
+use sp_runtime::traits::IdentityLookup;
+use sp_runtime::BuildStorage;
 
 use crate as vesting;
 
 pub type AccountId = u128;
 
+#[derive_impl(frame_system::config_preludes::TestDefaultConfig as frame_system::DefaultConfig)]
 impl frame_system::Config for Runtime {
-	type RuntimeOrigin = RuntimeOrigin;
```

Comment:
This formatting was kept as is from substrate's but I remember we do not want to keep the formatting anymore ? 🤔 
cc: @nazar-pc 

If not, can you also update `rustfmt.toml` please though I'm wondering why rustfmt automatically  format these files eve though its explicitly mentioned in the `rustfmt.toml`